app/migrations/Version20160205083520.php

```diff
@@ -30,7 +30,7 @@ public function up(Schema $schema)
         $this->addSql('CREATE TABLE sylius_product_review (id INT AUTO_INCREMENT NOT NULL, product_id INT NOT NULL, author_id INT DEFAULT NULL, title VARCHAR(255) NOT NULL, rating INT NOT NULL, comment LONGTEXT DEFAULT NULL, status VARCHAR(255) NOT NULL, created_at DATETIME NOT NULL, updated_at DATETIME DEFAULT NULL, INDEX IDX_C7056A994584665A (product_id), INDEX IDX_C7056A99F675F31B (author_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
         $this->addSql('ALTER TABLE sylius_product_review ADD CONSTRAINT FK_C7056A994584665A FOREIGN KEY (product_id) REFERENCES sylius_product (id) ON DELETE CASCADE');
         $this->addSql('ALTER TABLE sylius_product_review ADD CONSTRAINT FK_C7056A99F675F31B FOREIGN KEY (author_id) REFERENCES sylius_customer (id)');
-        $this->addSql('ALTER TABLE sylius_product ADD average_rating DOUBLE PRECISION DEFAULT NULL');
+        $this->addSql('ALTER TABLE sylius_product ADD average_rating DOUBLE PRECISION NOT NULL');
```

Comment:
Can change it, but this is how doctrine would generate it on its own, and also how the mapping is set currently